### ISSUE TITLE
Failed to install 'cordova.plugin.googleplus':Error: Uh oh!

### ISSUE THREAD

**Issue by Asit1989** [EddyVerbruggen/cordova-plugin-googleplus#86]
Getting following error while installing app 

Error during processing of action! Attempting to revert...
Failed to install 'cordova.plugin.googleplus':Error: Uh oh!
ENOENT, no such file or directory 'E:\123\google\hello\platforms\android\local.p
roperties'
    at Object.fs.openSync (fs.js:439:18)
    at Object.fs.readFileSync (fs.js:290:15)
    at Object.createEditor (C:\Users\piyush.kukadiya.DGSL\AppData\Roaming\npm\no
de_modules\cordova\node_modules\cordova-lib\node_modules\properties-parser\index
.js:328:39)
    at getProjectSdkDir (C:\Users\piyush.kukadiya.DGSL\AppData\Roaming\npm\node_
modules\cordova\node_modules\cordova-lib\src\plugman\platforms\android.js:36:45)

    at module.exports.framework.install (C:\Users\piyush.kukadiya.DGSL\AppData\R
oaming\npm\node_modules\cordova\node_modules\cordova-lib\src\plugman\platforms\a
ndroid.js:138:35)
    at Object.ActionStack.process (C:\Users\piyush.kukadiya.DGSL\AppData\Roaming
\npm\node_modules\cordova\node_modules\cordova-lib\src\plugman\util\action-stack
.js:67:25)
    at handleInstall (C:\Users\piyush.kukadiya.DGSL\AppData\Roaming\npm\node_mod
ules\cordova\node_modules\cordova-lib\src\plugman\install.js:575:20)
    at C:\Users\piyush.kukadiya.DGSL\AppData\Roaming\npm\node_modules\cordova\no
de_modules\cordova-lib\src\plugman\install.js:335:28
    at _fulfilled (C:\Users\piyush.kukadiya.DGSL\AppData\Roaming\npm\node_module
s\cordova\node_modules\q\q.js:787:54)
    at self.promiseDispatch.done (C:\Users\piyush.kukadiya.DGSL\AppData\Roaming\
npm\node_modules\cordova\node_modules\q\q.js:816:30)
Error: Uh oh!
ENOENT, no such file or directory 'E:\123\google\hello\platforms\android\local.p
roperties'
    at Object.fs.openSync (fs.js:439:18)
    at Object.fs.readFileSync (fs.js:290:15)
    at Object.createEditor (C:\Users\piyush.kukadiya.DGSL\AppData\Roaming\npm\no
de_modules\cordova\node_modules\cordova-lib\node_modules\properties-parser\index
.js:328:39)
    at getProjectSdkDir (C:\Users\piyush.kukadiya.DGSL\AppData\Roaming\npm\node_
modules\cordova\node_modules\cordova-lib\src\plugman\platforms\android.js:36:45)

    at module.exports.framework.install (C:\Users\piyush.kukadiya.DGSL\AppData\R
oaming\npm\node_modules\cordova\node_modules\cordova-lib\src\plugman\platforms\a
ndroid.js:138:35)
    at Object.ActionStack.process (C:\Users\piyush.kukadiya.DGSL\AppData\Roaming
\npm\node_modules\cordova\node_modules\cordova-lib\src\plugman\util\action-stack
.js:67:25)
    at handleInstall (C:\Users\piyush.kukadiya.DGSL\AppData\Roaming\npm\node_mod
ules\cordova\node_modules\cordova-lib\src\plugman\install.js:575:20)
    at C:\Users\piyush.kukadiya.DGSL\AppData\Roaming\npm\node_modules\cordova\no
de_modules\cordova-lib\src\plugman\install.js:335:28
    at _fulfilled (C:\Users\piyush.kukadiya.DGSL\AppData\Roaming\npm\node_module
s\cordova\node_modules\q\q.js:787:54)
    at self.promiseDispatch.done (C:\Users\piyush.kukadiya.DGSL\AppData\Roaming\
npm\node_modules\cordova\node_modules\q\q.js:816:30)

**Comment by ajoyoommen** [comment]
Same here

**Comment by EddyVerbruggen** [comment]
Hey guys, just released the 4.0.0 version to npm. I'm not sure what the cause of your error was, but this new release should work better across  various Cordova versions. 

```
$ cordova plugin add cordova-plugin-googleplus --variable REVERSED_CLIENT_ID=myreversedclientid
```

You can read about how to obtain the `REVERSED_CLIENT_ID` in the [readme](https://github.com/EddyVerbruggen). It makes the `iosApiKey` obsolete.
